docker-for-azure/upgrade.md

```diff
@@ -42,6 +42,18 @@ $ docker run \
   docker4x/upgrade-azure:17.06.0-ce-azure1
 ```
 
+If you are already on a version more recent than 17.06.1 CE or 17.07.0 CE, you can use the below script to initiate the upgrade:
+
+```bash
+$ upgrade.sh YY.MM.X-ce-azureC
+```
+
+For example, to upgrade from 17.06.1 CE edge to 17.07.0 CE edge, you can use:
```

Comment:
This example upgrades Docker for Azure from 17.06.1 CE Edge to 17.07.0 CE Edge:

Also, you should probably make it clear where the `upgrade.sh` script is expected to be, rather than just calling it from the current working directory.